dotcom-rendering/scripts/webpack/server.js

```diff
@@ -32,38 +32,31 @@ module.exports = () => ({
 	module: {
 		rules: [
 			{
-				test: /(\.tsx|\.js|\.ts)$/,
-				exclude: {
-					and: [/node_modules/],
-					not: [/@guardian/, /dynamic-import-polyfill/],
+				test: /\.js?$/,
+				loader: 'esbuild-loader',
+				exclude: module.exports.babelExclude,
```

Comment:
I don't think `module.exports.babelExclude` is available in this module?

Might get a small perf boost if the previous excludes are added back.